solution.txt

```diff
@@ -0,0 +1,260 @@
+/*
+Exercise 1:
+  - Define an empty array named foods
+*/
+
+// Exercise 1 has been completed for you...
+
+const foods = [];
+
+console.log('Exercise 1 Result:\n', foods);
+
+/*
+Exercise 2:
+  - Add the strings 'pizza' & 'cheeseburger' to the foods array such that 'pizza' comes before 'cheeseburger'.
+*/
+
+// Complete Exercise 2 below...
+
+foods.push('pizza', 'cheeseburger');
+
+console.log('Exercise 2 Result:\n',  foods);
+
+/*
+Exercise 3:
+  - Add the string 'taco' to the foods array so that 'taco' is the first food in the array.
+*/
+
+// Complete Exercise 3 below...
+
+foods.unshift('tacos');
+
+console.log('Exercise 3 Result:\n', foods);
+
+/*
+Exercise 4:
+  - Access the string 'pizza' (based upon its known position) in the foods array and assign to a variable named favFood.
+*/
+
+// Complete Exercise 4 below...
+
+//var favFood = foods.indexOf('pizza');
+
+var favFood = foods.slice(foods.indexOf('pizza'),foods.indexOf('pizza') + 1);
+favFood = favFood.toString();
+
+
+console.log('Exercise 4 Result:\n', favFood);
+
+/*
+Exercise 5:
+  - Insert the string 'tofu' in the foods array between 'pizza' & 'cheeseburger'
+*/
+
+// Complete Exercise 5 below...
+
+foods.splice(foods.indexOf('pizza') + 1, 0,'tofu');
```

Comment:
nice! I like how you made this solution programmatic, so that no matter where `pizza` is, you always insert `tofu` after it. 👍